localisation/wc_religions.csv

```diff
@@ -252,9 +252,13 @@ CAPITALIST;Capitalist;Capitaliste;;;;;;;;;;;;x;
 pandaren_religion_group;Chi;chi;;;;;;;;;;;;x;
 lightbound;Lightbound;;;;;;;;;;;;;x;
 lightbound_DESC;A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly.;;;;;;;;;;;;;x;
-shadowlander;Shadowlander;;;;;;;;;;;;;x;
-shadowlander_DESC;The Shadowlands are a dimension existing between worlds, where the souls of the mortal creatures go when they die. Their origins remain uncertain, but they have existed ever since mortal life first arose in the physical universe and seemingly span all worlds in existence. When a soul is called from the afterlife, it is free to roam the material realm if it is not restrained within moments of its arrival. Death knights, necromancers, and undead as a whole share a strong connection with the realm.;;;;;;;;;;;;;x;
-THE_FEL;the Fel;;;;;;;;;;;;;x;
+throsic;Throsic;;Throsisch;;;;;;;;;;;x;
+throsic_DESC;Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare, that they later named Thros, where they cultivated a potent form of death magic with bastardized druidic elements. This form of magic resulted from numerous attempts to bewitch the souls of fallen Drust into constructs in order to use against human settlers. Following this, the Throsic form of necromancy prefers binding the souls of the dead into durable constructs, while the traditional form of corpse reanimation is rarer. Its influence towards the wildlife and the living is usually perpetuated by the native covens. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through a ritualistic act of slitting their own throat in ultimate devotion to their horrific masters.;;Viele Geister der gefallenen Drust wurden innerhalb des smaragdgr�nen Albtraums, in eine Nebenebene verbannt, die sie sp�ter Thros nannten, wo sie eine m�chtige Form der Todesmagie mit verderbten druidischen Elementen erschufen. Diese Form der Magie entstand aus zahlreichen Versuchen, die Seelen Gefallener in Konstrukte zu binden, um sie gegen Siedler einzusetzen. Die throsische Form der Nekromantie bindet die Seelen der Toten vorzugsweise in dauerhafte Konstrukte, w�hrend die traditionelle Form der Totenerweckung seltener ist.  Ihr Einfluss auf die Tierwelt und die Lebenden wird gew�hnlich von den einheimischen Zirkeln aufrechterhalten. Die treuesten unter den throsischen Hexen k�nnen durch einen rituellen Akt, bei dem sie sich in �u�erster Hingabe an ihre schrecklichen Herren in einem rituellen Akt selbst die Kehle durchschneiden, auch zu Matronen, einer k�rperlosen, Banshee-�hnlichen Kreatur, aufsteigen.;;;;;;;;;;;x;
```

Comment:
"Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare that they later named Thros, where they"
removed the first comma.
"Drust into constructs in order to use against human settlers." should be either
"Drust into constructs to use against human settlers." or "Drust into constructs in order to use/utilize them against human settlers."
What does "Its influence towards the wildlife and the living is usually perpetuated by the native covens." mean?

Reply:
Btw, is it only against human settlers? Can't other races get there, too?

Reply:
That's for the writer to decide.

Reply:
Human settlers is written in historic context, the Drust waged war against the Gilneans that attempted to settle the island before they were ultimately banished to Thros and/or Ardenweald.